setup.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
 
-# Start ignoring PyImportSortBear as imports below may yield syntax errors
+# Start ignoring PyImportSortBear and PycodestyleBear* as imports below may
```

Comment:
I don't know why. Someone else told me how to put it and it seems like without the *, it doesn't work.